### PR TITLE
MTV-1385: Release Notes for MTV 2.6.6

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA 

* [MTV-1385](https://issues.redhat.com/browse/MTV-1385)

### PREVIEW

* [MTV 2.6.6 release notes](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-6.html)

    * [Resolved issues 2.6.6](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-6.html#resolved-issues-26-6_release-notes)
    
    * [Known issues](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-6.html#known-issues-26_release-notes)

### CODE REVIEW

File: documentation/modules/rn-2.6.adoc [hunk 1]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
...
+
+.Runtime error: invalid memory address or nil pointer dereference
+
+In earlier releases of {project-short}, there was a runtime error of `invalid memory address` or `nil pointer dereference` caused by a pointer that was nil, and there was an attempt to access the value that it points to. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1353[(MTV-1353)]
+
+.All Plan and Migration pods scheduled to same node causing the node to crash
+
+In earlier releases of {project-short}, the scheduler could place all migration pods on a single node. When this happened, the node ran out of the resources. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1354[(MTV-1354)]
+
+.Empty bearer token is sufficient for authentication
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:155]: As this ticket is marked private, should we remove it from the release notes

File: documentation/modules/rn-2.6.adoc [hunk 2]
```diff
@@ -316,6 +331,10 @@ For a complete list of all resolved issues in this release, see the list of link
 
 This release has the following known issues:
 
+.Warm migration and remote migration flows are impacted by multiple bugs
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:334]: * Do you also want an admonition adding in [Warm migrations docs](https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.6/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#warm-migration_mtv)?

File: documentation/modules/rn-2.6.adoc (hunk 1 above)
Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:155]: It's a CVE, how do we document CVEs?

File: documentation/modules/rn-2.6.adoc (hunk 2 above)
Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:334]: Can we make this a INFO or NOTE box?

File: documentation/modules/rn-2.6.adoc (hunk 1 above)
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc:155]: If you want me to include the CVE, then we can.

Personally, I would prefer to include them with a link to the Red Hat CVE library

| CVE example |
| ------------------- |
| 
image |

What do you think? If you want me to include it, please ping me the CVE number and I will include.

File: documentation/modules/rn-2.6.adoc (hunk 2 above)
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc:335]: @fabiand . Yes, of course (makes sense).

 There are five admonition types:

* NOTE
* TIP
* IMPORTANT
* CAUTION
* WARNING

image

Added as a WARNING, but it is a single attribute change. So which would you prefer?

File: documentation/modules/rn-2.6.adoc [hunk 3]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
...
+
+In earlier releases of {project-short}, there was a runtime error of `invalid memory address` or `nil pointer dereference` caused by a pointer that was nil, and there was an attempt to access the value that it points to. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1353[(MTV-1353)]
+
+.All Plan and Migration pods scheduled to same node causing the node to crash
+
+In earlier releases of {project-short}, the scheduler could place all migration pods on a single node. When this happened, the node ran out of the resources. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1354[(MTV-1354)]
+
+.Empty bearer token is sufficient for authentication
+
+In earlier releases of {project-short}, an empty bearer token was sufficient for authentication. This issue has been resolved in {project-short} 2.6.6. link:https://issues.redhat.com/browse/MTV-1377[(MTV-1377)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:157]:
```suggestion
In earlier releases of {project-short}, a vulnerability was found in the Forklift Controller.  There is no verification against the authorization header, except to ensure it uses bearer authentication. Without an authorization header and a bearer token, a `401` error occurs. The presence of a token value provides a 200 response with the requested information. This issue has been resolved in {project-short} 2.6.6. 

For more details, see link:https://access.redhat.com/security/cve/cve-2024-8509[(CVE-2024-8509)].
```

File: documentation/modules/rn-2.6.adoc (hunk 1 above)
Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:155]: Let's take what you prefer :)

File: documentation/modules/rn-2.6.adoc (hunk 2 above)
Comment (fabiand) [reply documentation/modules/rn-2.6.adoc:337]: this looks to me.

**Comment by fabiand** [comment]
lgtm!

**Pull request merged**